OpenEdXMobile/src/main/java/org/edx/mobile/module/analytics/Analytics.java

```diff
@@ -873,6 +883,8 @@ void trackValuePropLearnMoreTapped(@NonNull String courseId, @Nullable String as
         // Course Celebration Modal
         String COURSE_SECTION_COMPLETION_CELEBRATION = "Celebration: First Section Opened";
         String CELEBRATION_SOCIAL_SHARE_CLICKED = "Celebration: Social Share Clicked";
+        // Resume Course Banner
+        String RESUME_COURSE_BANNER = "Resume Course";
```

Comment:
The event name is `Resume Course Tapped` as per the story.

Reply:
Done